packages/flutter_tools/test/general.shard/build_system/targets/common_test.dart

```diff
@@ -75,6 +75,15 @@ void main() {
     fileSystem.file('.dart_tool/package_config.json')
       ..createSync(recursive: true)
       ..writeAsStringSync('{"configVersion": 2, "packages":[]}');
+    androidEnvironment.buildDir.childFile('native_assets.yaml')
+      ..createSync(recursive: true)
+      ..writeAsStringSync('''
+format-version:
+  - 1
+  - 0
+  - 0
+native-assets: {}
+''');
```

Comment:
Looks like this would benefit from a helper so we're not copy/pasting this a bunch of times.